omigami/data/data_models.py

```diff
@@ -1,18 +1,33 @@
-# TODO: rename as data_models
-
-from typing import Iterable, List
+from concurrent.futures._base import Future
+from typing import Iterable, List, Dict, Union
 from dataclasses import dataclass
-from omigami.types import NumpyArray
+from omigami.data import NumpyArray
```

Comment:
I actually liked the types module a lot ;)

Reply:
:]